docs/introduction/zoned-storage.md

```diff
@@ -33,85 +33,88 @@ possible to meet expanding data needs, and SMR technology requires the zoned
-
-2. Managing zoned storage directly from the application layer is a valid
-   approach but can be difficult to implement, particularly for use cases where
-   several device features must be combined (e.g. Using I/O priorities).
-   Application level support can be simplified by relying on the
-   kernel ZBD support which allows accessing and managing zoned devices using
-   regular POSIX system calls. While this does not remove the zoned device
-   access constraints from the application scope, implementation of sequential
-   write streams and zone management can be simplified.
-
-3. If a user lacks the software control at the application level while retaining
-   control over Linux kernel version choices, more advanced kernel support
-   features such as ZBD compliant file systems can be used to hide zoned storage
-   access constraints from the application. Users may also rely on
-   a device mapper driver exposing zoned storage devices as regular block
-   devices. With such solution, existing file systems can be used without any
-   modification.
+There are several ways to design a system application for zoned storage.  The
+way that is right for you depends upon your system's structure and your ability
+to modify the application layer and your ability to choose a particular Linux
+kernel version.
+
+1. **Users that own software applications and do not depend on the operating
+   system and file system to control the device** can directly use the device
+   interface protocol to issue zone management commands by using a
+   **passthrough interface**. In these cases, the applications must be
+   rewritten with the new command sets and they must ensure that all data
+   streams are sequential. The [*libzbc* library](../tools/libzbc) provides
+   functions to help with this approach.
+
+2. **Managing zoned storage directly from the application layer** is a valid
+   approach but **can be difficult to implement**. This is true particularly in
+   use cases where several device features must be combined (for example, when
+   I/O priorities are used). Application level support can be simplified by
+   relying on the kernel ZBD support which allows accessing and managing zoned
+   devices using regular POSIX system calls. While this does not remove the
+   zoned device access constraints from the application scope, implementation
+   of sequential write streams and zone management can be simplified.
+
+3. If **a user lacks software control at the application level but retains
+   control over the choice of the version of the Linux kernel, more advanced
+   kernel support features such as ZBD compliant file systems can be used to
+   hide zoned storage access constraints from the application**. Users may also
+   rely on a device mapper driver that exposes zoned storage devices as regular
+   block devices. **If this solution is used, existing file systems can be used
+   without any modification**.
```

Comment:
I want to note here that the emphasized text (that looks like this in the markdown: **text**) is rendered correctly in vim on my local machine, but is not rendered correctly here in github's code browser.

Weird.

Reply:
I am not convinced these bold emphasis changes bring any value. And there is a bug in bullet (3) (missing closing **).

Reply:
I modified this when applying